evalai/utils/challenges.py

```diff
@@ -160,3 +160,128 @@ def get_challenge_count(is_host=False, is_participant=False):
+        phases = response_json["results"]
+        print_phases(phases)
+
+
+def print_phase_details(phase):
+    """
+    Print details of a particular phase
+    """
+    phase_title = "\n{}".format(style(phase["name"], bold=True, fg="green"))
+    challenge_id = "Challenge ID: {}".format(style(
+                                      str(phase["challenge"]),
+                                      bold=True, fg="blue"))
+    phase_id = "Phase ID: {}\n\n".format(style(str(phase["id"]),
+                                         bold=True, fg="blue"))
+
+    title = "{} {} {}".format(phase_title, challenge_id, phase_id)
+
+    cleaned_desc = BeautifulSoup(phase["description"], "lxml").text
+    description = "{}\n".format(cleaned_desc)
+
+    start_date = "Start Date : " + style(phase["start_date"].split("T")[0],
+                                         fg="green")
+    start_date = "\n{}\n".format(style(start_date, bold=True))
+
+    end_date = "End Date : " + style(phase["end_date"].split("T")[0], fg="red")
+    end_date = "\n{}\n".format(style(end_date, bold=True))
+
+    max_submissions_per_day = style("\nMaximum Submissions per day : {}\n".format(str(
+                                        phase["max_submissions_per_day"])),
+                                    bold=True)
+
+    max_submissions = style("\nMaximum Submissions : {}\n".format(
+                                str(phase["max_submissions"])), bold=True)
+
+    codename = style("\nCode Name : {}\n".format(
+                                          phase["codename"]), bold=True)
+
+    leaderboard_public = style("\nLeaderboard Public : {}\n".format(
+                                phase["leaderboard_public"]), bold=True)
+
+    is_active = style("\nActive : {}\n".format(
+                                          phase["is_active"]), bold=True)
+
+    is_public = style("\nPublic : {}\n".format(
+                                          phase["is_public"]), bold=True)
+
+    phase = "{}{}{}{}{}{}{}{}{}{}".format(title, description, start_date,
+                                          end_date,
+                                          max_submissions_per_day,
+                                          max_submissions,
+                                          leaderboard_public,
+                                          codename, is_active, is_public)
+    echo(phase)
+
+
+def get_phase_details(challenge_id, phase_id):
+    """
+    Gets the phase details of a particular challenge phase.
+    """
+    url = Urls.phase_details.value
+    url = "{}{}".format(API_HOST_URL, url)
+    url = url.format(challenge_id, phase_id)
+    headers = get_headers()
+
+    try:
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        echo(style("Error: " + response.json()['error'], fg="red", bold=True))
+        sys.exit(1)
+    except requests.exceptions.RequestException as err:
+        echo(err)
```

Comment:
@varunagarwal @deshraj I think we should this error somewhere and display a message to user like `You are not connected to the internet. Please check your connection` What do you say?